static/src/javascripts/projects/common/modules/tailor/tailor.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import fetchData from 'common/modules/tailor/fetch-data';
+
+/**
+ * Given a response from tailor, we see if the response has a survey suggestion, and if so return the first
+ * survey suggestion (there should only ever be one, but just in case).
+ *
+ * @returns {Promise.<any>}
```

Comment:
Glad you spotted that, I was thinking the same thing 😄  Then I second-guessed myself... "But what if, one day, we start using JSDoc?" Perhaps I'll kill the comments for now and we can reinstate them if we need them in the future.